app/views/visns/_show_table_row.html.erb

```diff
@@ -1,10 +1,30 @@
 <% facilities.each do |facility| %>
+  <% path = facility.class.name === "ClinicalResourceHub" ? "/crh/" + facility.visn.number.to_s : va_facility_path(facility)  %>
+  <% if facility.class.name === "ClinicalResourceHub" %>
+    <% created_count = crh_created_practices_count(facility.visn_id) %>
+    <% adopted_count = crh_adopted_practices_count(facility.visn_id) %>
+  <% else %>
+    <% created_count = facility.practices_created_count %>
+    <% adopted_count = facility.practices_adopted_count %>
```

Comment:
I think we can use ternary statements here instead of setting the variable twice. We can say something like 
```
created_count = is_crh ? crh_created_practices_count(facility.visn_id) : facility.practices_created_count
```

Reply:
Yes much better...